src/analyze.ts

```diff
@@ -179,7 +178,10 @@ interface TokenLocation {
 }
 
 function _isExportStatement (exportsLocation: TokenLocation[], exp: ESMExport) {
-  return exportsLocation.some(location => exp.start <= location.start && exp.end >= location.end)
+  return exportsLocation.some(location =>
+    (exp.start <= location.start && location.start <= exp.end) ||
+    (exp.start <= location.end && location.end <= exp.end)
```

Comment:
Previous condition (token _within_ exp) seems keep tests pass. Why we need to change it to (token start within match || token end within match)?

Reply:
Made it into a comment to marge fix asap. fbf5c419df1d938286934b745d4be14104d3514d. Would be happy to discuss more and change checking strategy if you think there is a case needs this check.

Reply:
That was my initial assumption of the cause - yes I think we don't need to change it for this case.